collamat_kareen/Activity3/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE HTML>
+<html>
+	<head>
+		<title>Resume of Kareen Collamat</title>
+		<link rel="stylesheet" href="style.css">
+	</head>
+	<body>
+		<div class="container">
+			<center>
+			<div class="upperfold">	
+				<div class="upperfold-content">
+					<div class="image-cropper">
+						<img src="images/img1.png">
+					</div>
+					<h1>Kareen Collamat</h1>
+					<hr />
+					<h2>Fullstack Web Developer</h2>
+					<p>
+					<a href="tel:+639306378788" target="_blank">0930-637-8788</a>
+					| <a href="mailto:kaorusan2019@gmail.com" target="_blank">kaorusan2019@gmail.com</a>
+					| <a href="https://www.linkedin.com/in/kareen-san-952b95215/" target="_blank">My LinkedIn</a> 
+					| <a href="https://www.facebook.com/kaorusan1223/" target="_blank">My Facebook</a>
+				</div>
+			</div>
+			</center>
+			<div class="content">
+				<div class="one">
+					<h2>Educational Attainment</h2>
+					<h3>Highschool</h3>
+					Letran de Davao High School<br />					
+					2007
+					<h3>Elementary</h3>
+					Briz Tagum Elementary School
+				</div>
+				<div class="two">
+					<h2>Work Experience</h2>
+					<h3>Front End Developer</h3>
+					Ocean Swell Digital<br />					
+					2021 - Present
+					<h3>Freelance Front End Developer</h3>
+					PBN Sites, Local Pasty Shop, On Call Maintenance<br />					
+					2018 - 2021
+				</div>
+			</div>
+			<div style="clear:both;"></div>
+			<div class="content2">
+				<div class="three">
```

Comment:
same with this, do not use variable class name with numbers

Reply:
Updated, sir.